src/explorer-components/FilterNav/index.js

```diff
@@ -0,0 +1,290 @@
+import React from 'react';
+import styles from './FilterNav.module.scss';
+import PropTypes from 'prop-types';
+import {
+  Search,
+  Text,
+  Button,
+  Checkbox,
+  RadioGroup,
+  SearchSelect
+} from 'components';
+import {
+  curry,
+  each,
+  map as fpMap,
+  indexOf,
+  reduce as fpReduce,
+  throttle
+} from 'lodash';
+import {
+  toggleFromParam,
+  pushToParam,
+  removeFromParam,
+  setParam
+} from 'utils/locationHelpers';
+import appConfig from 'public-modules/config';
+
+const map = fpMap.convert({ cap: false });
+const reduce = fpReduce.convert({ cap: false });
+
+const FilterNav = props => {
+  const {
+    // user props
+    resetableFilters,
+    config,
+    position,
+
+    // current state
+    currentDifficultyFilters,
+    currentPlatformFilters,
+    currentSearchValue,
+    currentSortFilter,
+    currentStageFilter,
+    currentSelectedCategories,
+
+    availableCategories,
+
+    // defaults
+    defaultStageFilters,
+    defaultPlatforms,
+
+    // actions
+    addCategoryFilter,
+    addPlatformFilter,
+    filterReseter, // dispatches action to reset a specific filter
+    setSearch,
+    setSort,
+    toggleStageFilter,
+    toggleDifficultyFilter,
+    togglePlatformFilter,
+    removeCategoryFilter,
+
+    history,
+    location,
+    batch
+  } = props;
+
+  const sortOptions = [
+    { value: 'bounty_created', label: 'Most Recent' },
+    { value: 'usd_price', label: 'Value: High to Low' },
+    { value: 'deadline', label: 'Expiry' }
+  ];
+
+  const platforms = appConfig.platform.split(',');
+
+  const stages = reduce(
+    (acc, value, key) => {
+      if (value) acc.push(key);
+      return acc;
+    },
+    [],
+    defaultStageFilters
+  ).join('%2C');
+
+  const rootLocationParams =
+    location.search ||
+    `?bountyStage=${stages}&platform=${defaultPlatforms.join(',')}`;
+
+  const baseAction = curry((queryModifier, key, value, action) => {
+    value = Array.isArray(value) ? value : [value];
+    const queryParams = queryModifier(rootLocationParams, key, value.join(','));
+    history.push(location.pathname + queryParams);
+    action(...value);
+  });
+
+  const toggleAction = baseAction(toggleFromParam);
+  const pushAction = baseAction(pushToParam);
+  const removeAction = baseAction(removeFromParam);
+  const setAction = baseAction(setParam);
+
+  const toggleStageFilterAction = stage => toggleAction('bountyStage', stage, toggleStageFilter); // prettier-ignore
```

Comment:
Why the prettier ignore?

Reply:
because it really bothers me when it =
   breaks those statement into 2 lines at `=` sign

let me know if i should just remove

Reply:
just remove the rule... :)